src/mongo/commands.ts

```diff
@@ -30,7 +31,21 @@ export class MongoCommands {
 				throw new Error('Please connect to the database first');
 			}
 			const result = await database.executeCommand(command);
-			await util.showNewFile(result, extensionPath, 'result', '.json', activeEditor.viewColumn + 1);
+			const parsed = JSON.parse(result);
+			if (command.name === 'find' || command.name === 'findOne') {
+				const db = await database.getDb();
+				let node: MongoCollectionNode | MongoDocumentNode;
+				if (command.name === 'find') {
+					node = new MongoCollectionNode(db.collection(command.collection), database, parsed, command.arguments);
```

Comment:
Call node.getChildren() here right after you create the node.
This method is async